src/style.scss

```diff
@@ -1,14 +1,11 @@
-body {
-  font-family: 'Roboto', sans-serif;
-  margin: 0;
-  padding: 0;
-}
 
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap');
```

Comment:
Can we use the open source version of Roboto without using google's api?

https://opensource.googleblog.com/2015/05/roboto-googles-signature-font-is-now.html

Reply:
We might have problems with it, I think much easier would be to use some highly available font with a set of failbacks.

Reply:
Sure, we'll discuss this with the designers as well.